__tests__/api.test.js

```diff
@@ -65,3 +64,40 @@ describe("GET: /api", () => {
       });
   });
 });
+describe("GET: /api/articles/1", () => {
+  test("200: should return an article by its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.article;
+
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("400: ERROR - responds with the error if the data type for id is incorrect", () => {
+    return request(app)
+      .get("/api/articles/nonsense")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+  test("404: ERROR - responds with the Bad Request", () => {
+    return request(app)
+      .get("/api/articles/9999")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
```

Comment:
The status code is correct, however, your message implies this is a bad request. This is an article that doesn't exist so wouldn't be a bad request. Please do change the message